data/glossary.yml

```diff
@@ -138,6 +167,47 @@
     - title: Kyū - Wikipedia
       url: https://en.wikipedia.org/wiki/Ky%C5%AB
 
+- term: Language versions
+  description: |
+    A kata may be available in one or more languages (javascript, python, coq, ...), and for each language there may be different versions like Node 8 or Node 10 for JS, python 3.4.3 or 3.6, ... You can choose the language _and_ the language version in the trainer of a kata (amongst the available ones only).
+
+- term: Leaderboards
+  description: Leaderboards are grouping different users according to different criteria.
+  links:
+    - title: Overall top 500
+      url: https://www.codewars.com/users/leaderboard
+    - title: Top 500 by completed Katas
+      url: https://www.codewars.com/users/leaderboard/kata
+    - title: Top 500 by authored Katas and translations
+      url: https://www.codewars.com/users/leaderboard/authored
+    - title: Your personnal leaderboard (with your allies, in the dashboard)
+      url: https://www.codewars.com/dashboard
+
+- term: Markdowns (formatting)
+  description: |
+    CodeWars is responsive to `markdowns`. They allow you to format code, descriptions or comments so that thay are properly readable for other users.
```

Comment:
```suggestion
    Codewars is responsive to `markdowns`. They allow you to format code, descriptions or comments so that thay are properly readable for other users.
```